wled00/data/index.js

```diff
@@ -579,10 +579,11 @@ function populateSegments(s)
 				<span class="checkmark schk"></span>
 			</label>
 			<div class="segname" onclick="selSegEx(${i})">
-				Segment ${i}
+				${inst.n ? inst.n : "Segment "+i}
 			</div>
 			<i class="icons e-icon flr ${expanded[i] ? "exp":""}" id="sege${i}" onclick="expand(${i})">&#xe395;</i>
 			<div class="segin ${expanded[i] ? "expanded":""}" id="seg${i}">
```

Comment:
There is a too big gap between segment top line and name field. Will decrease margin in css.

Reply:
Yes. I do have slightly modified CSS in my fork. Sorry.